index.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <!-- Navbar -->
+
+  <header>
+    <div class="nav-container">
+      <nav class="nav-bar">
+        <div class="logo">
+          <div class="logo-placeholder">
+            <a href="#">
+              <h1>Welcome</h1>
+            </a>
+          </div>
+        </div>
+        <div class="harmburger">
+          <img src="./images/Union.svg" alt="Toggle button to display hidden nav items">
+        </div>
+      </nav>
+    </div>
+  </header>
+  <section>
+    <div class="headline-intro">
+      <div class="header-content">
+        <p class="intro">Hey there. I'm Oyinlade <br>
+          <span class="hello-span">I'm a software developer</span></p>
+        <p class="bio"> I can help you build a product,feature or website. Look through some of my work and
+          experience! If you like what you see and have a project you need coded,
+          don't hestiate to contact me.</p>
+      </div>
+      <!-- Social Media links -->
+      <div class="social-media-icon">
+
+        <a href="#"> <img src="./images/twitter.png" alt=""></a>
+        <a href="#"> <img src="./images/Linkedin-icon.svg" alt=""></a>
+        <a href="#"><img src="./images/Messenger.png" alt=""></a>
+        <a href="#"><img src="./images/github.png" alt=""></a>
+        <a href="#"><img src="./images/two-fingers.svg" alt=""></a>
+      </div>
```

Comment:
- [ ] Kindly be noticed that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

Reply:
Thank you so much for the feedback, I will work on the corrections now aside from the changing of the responsiveness, for it might take some time to make it responsive on every screen size and I have limited time now. I will do that later.